missions/2PzD_Template_v3_0.VR/customization/geardefs/US/gearDefUSRegular.sqf

```diff
@@ -1,46 +1,46 @@
 //======================== US Uniform Definitions ========================
 
 //Uniform
-#define US_Uni_CPT              "fow_u_us_m41_01_private"
-#define US_Uni_LT               "fow_u_us_m41_01_private"
+#define US_Uni_CPT              "U_LIB_US_Private"
+#define US_Uni_LT               "U_LIB_US_Private"
 #define US_Uni_SGT              "U_LIB_US_Sergeant"
 #define US_Uni_CPL              "U_LIB_US_Corp"
-#define US_Uni_PFC              "fow_u_us_m41_01_private"
-#define US_Uni_PVT              "fow_u_us_m41_01_private"
-#define US_Uni_Med              "fow_u_us_m41_01_private"
-#define US_Uni_Tec5             "fow_u_us_m41_01_private"
+#define US_Uni_PFC              "U_LIB_US_Private"
+#define US_Uni_PVT              "U_LIB_US_Private"
+#define US_Uni_Med              "U_LIB_US_Private"
+#define US_Uni_Tec5             "U_LIB_US_Private"
 
 #define US_Uni_VCrew1           "U_LIB_US_Tank_Crew"
 #define US_Uni_VCrew2           "U_LIB_US_Tank_Crew2"
 
 //Vests
-#define US_Vest_Pistol          "fow_v_us_45"
+#define US_Vest_Pistol          "V_LIB_US_Vest_45"
 #define US_Vest_NCO_M1C         "fow_v_us_carbine_nco"
 #define US_Vest_NCO_M1T         "fow_v_us_thompson_nco"
-#define US_Vest_M1G             "fow_v_us_garand"
+#define US_Vest_M1G             "V_LIB_US_Vest_Garand"
 #define US_Vest_M1G_Frag        "fow_v_us_Grenade"
 #define US_Vest_M1G_Gren        "fow_v_us_grenade"
-#define US_Vest_M1C             "fow_v_us_carbine"
+#define US_Vest_M1C             "V_LIB_US_Vest_Carbine"
 #define US_Vest_M1C_Eng         "fow_v_us_carbine_eng"
-#define US_Vest_M1T             "fow_v_us_thompson"
-#define US_Vest_BAR             "fow_v_us_bar"
-#define US_Vest_MGA             "fow_v_us_asst_mg"
-#define US_Vest_Med             "fow_v_us_medic"
+#define US_Vest_M1T             "V_LIB_US_Vest_Thompson"
+#define US_Vest_BAR             "V_LIB_US_Vest_Bar"
+#define US_Vest_MGA             "V_LIB_US_Vest_M1919"
+#define US_Vest_Med             "V_LIB_US_Vest_Medic"
 
 //Headgear
 #define US_Helm_CPT             "H_LIB_US_Helmet_Cap"
 #define US_Helm_LT              "H_LIB_US_Helmet_First_lieutenant"
 #define US_Helm_2LT             "H_LIB_US_Helmet_Second_lieutenant"
-#define US_Helm_NCO_r           ["fow_h_us_m1_nco"],["fow_h_us_m1_nco_closed"],["fow_h_us_m1_nco_folded"]
-#define US_Helm_r               ["fow_h_us_m1"],["fow_h_us_m1"],["fow_h_us_m1"],["fow_h_us_m1_closed"],["fow_h_us_m1_closed"],["fow_h_us_m1_closed"],["fow_h_us_m1_folded"],["fow_h_us_m1_folded"],["fow_h_us_m1_folded"],["fow_h_us_m1_net"],["fow_h_us_m1_net"],["fow_h_us_m1_net"]
-#define US_Helm_Med_r           ["fow_h_us_m1_medic"],["fow_h_us_m1_medic"]
+#define US_Helm_NCO_r           "H_LIB_US_Helmet_NCO"
+#define US_Helm_r               ["H_LIB_US_Helmet"],["H_LIB_US_Helmet_Net_ns"],["H_LIB_US_Helmet_Net_os"]
+#define US_Helm_Med_r           "H_LIB_US_Helmet_Med"
```

Comment:
Same here with the "_r"

Reply:
how will this affect the Winter part of this? as that has several Classnames?